frontend/src/router.ts

```diff
@@ -305,4 +305,29 @@ export const routerBeforeEach: NavigationGuard = async (to, from, next) => {
   next();
 };
 
-router.beforeEach(routerBeforeEach);
+export const redirectSections: NavigationGuard = async (to, from, next) => {
+  if (_.get(to, "meta.activator", false)) {
+    const activatorChildren = getRootRouteChildren().find(
+      (route: RouteConfig) => `/${route.path}` === `${to.fullPath}/`
+    ).children;
```

Comment:
Can't we just use `to` instead of finding it by path?

> to: Route: the target Route Object being navigated to.

Reply:
The `to` object from the route guard does not contain children like the routes on the `router` object.